src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,18 @@
+rootLogger.level = INFO
```

Comment:
Also, as part of manual testing, did you try setting this to a finer level and see if you can get debug messages going? Although it does not seem the is much in terms of logging, you'd probably need to set up a proxy in order to get anything interesting in the output.

Reply:
yes it has been tested manually. See comment below.